control/veda_deliver_3play.py

```diff
@@ -94,6 +101,40 @@ def validate_media_url(self):
                 )
             )
 
+    def get_available_languages(self):
+        """
+        Gets available languages
+        """
+        response = requests.get(url=build_url(self.base_url, self.available_languages_url, apikey=self.api_key))
```

Comment:
Why we need to do this for every transcription request? Can we cache the response to avoid the request every time?